docusaurus/docs/key-concepts/ui-extensions.md

```diff
@@ -9,57 +9,74 @@ keywords:
+- **Plugin developer**: A developer working with the Grafana plugins ecosystem.
 
-Extension points facilitate the breaking down of silos between individual views, allowing users to quickly leverage data from their current context to take relevant actions. By providing extension points within your app, relevant extensions can easily offer new capabilities to your shared users.
+## I want to render extension content 
 
-Adding an extension point to your UI provides the following benefits:
+As a content user, you need to define an extension point in your plugin to be able to render content (links or components) made available by content providers.
 
-- Define the UI extension point once to enable multiple plugins to extend your UI with new functionality. You don't need any additional effort to provide functionality from more plugins in your UI.
-- Clean separation of concerns. Your application doesn't need to know anything about the plugin extending your UI.
-- Integration built for change. Since your application doesn't know anything about the internal workings of the plugin extending your UI, they are free to change their APIs without the risk of breaking the extension point UI.
-- Easy to bootstrap. If both apps are installed and enabled, then the extensions are automatically configured and displayed to the user. There is no need for either app to include custom logic to detect the presence of the other.
-- Extensions are fast. We pre-build the extensions registry at Grafana boot time which makes it fast to use while rendering the UI.
+### Why add an extension point?
 
-## Use cases
+Define extension points to add new capabilities:
 
-You can use extensions for the following scenarios:
+- An extension point allows other plugins to extend your UI with new functionality. You don't need any additional effort to provide functionality from other plugins in your UI.
+- Clean separation of concerns. Your application doesn't need to know anything about the plugin extending your UI.
+- Easy to bootstrap. If both apps are installed and enabled, then the extensions are automatically configured and displayed for your user. There is no need for either app to include custom logic to detect the presence of the other.
 
-- The user views a dashboard with historical data. By adding an extension point to this part of the UI, a machine learning app plugin can give the user the ability to create a forecast for that data directly from the panel.
-- The user views a firing alert. By adding an extension point to this part of the UI, an Incident app plugin can give the user the ability to create an incident directly from the alert view.
+### Where can I find the content rendered in my extension point?
```

Comment:
Maybe I am misunderstanding something, and in that case I am sorry. To me this heading ("Where can I find the content rendered in my extension point?") refers to how I could locate my extension point with any registered extensions on the UI, while below we are showing an example for an extension point. Is that intended?